tests/fft.hpp

```diff
@@ -214,3 +216,87 @@ static void test_fft()
     ASSERT_LE(criterion, epsilon);
     ASSERT_LE(criterion2, epsilon);
 }
+
+template <typename ExecSpace, typename MemorySpace, typename Tin, typename Tout, typename X>
+static void test_fft_norm(ddc::FFT_Normalization const norm)
+{
+    DDom<DDim<X>> const x_mesh
+            = DDom<DDim<X>>(ddc::init_discrete_space(DDim<X>::
+                                                             init(ddc::Coordinate<X>(-1. / 4),
+                                                                  ddc::Coordinate<X>(1. / 4),
+                                                                  DVect<DDim<X>>(2))));
+    ddc::Chunk _f = ddc::Chunk(x_mesh, Allocator<MemorySpace, Tin>());
+    ddc::ChunkSpan f = _f.span_view();
+    ddc::for_each(
+            policy<ExecSpace>(),
+            ddc::get_domain<DDim<X>>(f),
+            DDC_LAMBDA(DElem<DDim<X>> const e) { f(e) = static_cast<Tin>(1); });
+
+    ddc::init_fourier_space<X>(x_mesh);
+    DDom<DFDim<ddc::Fourier<X>>> const k_mesh
+            = ddc::FourierMesh(x_mesh, is_complex<Tin>::value && is_complex<Tout>::value);
+
+    ddc::Chunk _f_bis = ddc::Chunk(ddc::get_domain<DDim<X>>(f), Allocator<MemorySpace, Tin>());
+    ddc::ChunkSpan f_bis = _f_bis.span_view();
+    ddc::deepcopy(f_bis, f);
+
+    ddc::Chunk _Ff = ddc::Chunk(k_mesh, Allocator<MemorySpace, Tout>());
```

Comment:
`_Ff` is a reserved keyword in C++, we do not use underscores at the start of variable names for this reason .

In fact we should get back our coding standard that we lost in the split with gyselalib++ https://github.com/gyselax/gyselalibxx/blob/main/doc/CODING_STANDARD.md

Reply:
@blegouix you can find more information in this post https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier

Reply:
we should check if clang-tidy is able to detect this ?

Reply:
Is there a convention to name the chunk _FFf in `ddc::ChunkSpan FFf = _FFf.span_view();` ?

Reply:
In the MR #158 I used `FFf_alloc`, I don't think we have a clear convention about that naming scheme.

Reply:
I propose to merge this first and I will align MR #158 afterward